src/poetry/utils/env/virtual_env.py

```diff
@@ -133,3 +135,14 @@ def temp_environ(self) -> Iterator[None]:
 
     def _updated_path(self) -> str:
         return os.pathsep.join([str(self._bin_dir), os.environ.get("PATH", "")])
+
+    def includes_system_site_packages(self) -> bool:
+        pyvenv_cfg = self._path / "pyvenv.cfg"
+        return "include-system-site-packages = true" in pyvenv_cfg.read_text()
```

Comment:
Maybe, we should make this a `cached_property` since this should not change during one invocation of poetry. Now, we are reading `pyvenv.cfg` once for each package.